src/NumFieldOrd/NfRelOrd/NfRelOrd.jl

```diff
@@ -564,21 +549,14 @@ function maximal_order_via_simple(m::NumFieldMor{<:NfRel, <:NfRelNS})
 end
 
 function maximal_order_via_relative(K::AnticNumberField, m::NfToNfRel)
-  try
-    O = _get_maximal_order(K)
-    return O
-  catch e
-    if !isa(e, AccessorNotSetError)
-      rethrow(e)
-    end
-  end
-  L = codomain(m)
-  OL = maximal_order(L)
-  B = absolute_basis(OL, L)
-  OK = Order(K, [ m\b for b in B ], check = false, isbasis = true)
-  OK.ismaximal = 1
-  _set_maximal_order(K, OK)
-  return OK
+  return get_attribute!(K, :maximal_order) do
+    L = codomain(m)
+    OL = maximal_order(L)
+    B = absolute_basis(OL, L)
+    OK = Order(K, [ m\b for b in B ], check = false, isbasis = true)
+    OK.ismaximal = 1
+    return OK
+  end::order_type(OK)
```

Comment:
Does this really work? Isn't `OK` out of scope here?

Reply:
Maybe 
```suggestion
  end::order_type(K)
```
But there were bo test failures, so either a test should be added or I am misunderstanding something badly

Reply:
Yes, there was a test failure, see https://github.com/thofma/Hecke.jl/runs/5009676182?check_suite_focus=true